pcsx2/ps2/BiosTools.h

```diff
@@ -28,6 +28,23 @@ struct BiosDebugInformation
 	u32 threadListAddr;
 };
 
+static const char* BiosZoneStrings[] {
+	"T10K",
+	"Test",
+	"Japan",
+	"USA",
+	"Europe",
+	"HK",
+	"Free",
+	"China",
+	nullptr
+};
+
+static const char* BiosZoneBytes[]
+{
+	"T", "X", "J", "A", "E", "H", "P", "C", 0x0
```

Comment:
Strictly speaking this should be `nullptr` on the end (since it's a pointer array).